### ISSUE TITLE
Add a funny little but formal tongue to LMMS

### ISSUE THREAD

**Issue by YesImAsh** [LMMS/lmms#7376]
### Enhancement Summary

Its essentially a formant/vocal synth which can be controlled using Midi and it has the ability to control how the target plugin sounds

The reason why this should be implemented is because Formants in LMMS are either hard to use or to even make at all

Even VeSTige Cant do it easily let alone at all

### Implementation Details / Mockup

Step 1. Add target synth (Triple oscillator for example; use a single saw)
Step 2. Add Formant synth
Step 3. In FX, Triple osc will have a very special effect where it assigns to the formant synth
Step 4. Create a melody in Triple osc
Step 5. Using the formant, assign each different sound to different midi notes (maybe add presets like English, Spanish, French etc)
Step 6. Enable ghost mode for triple osc
Step 7.  Go to formant, and place the notes to where you assigned the sound to (Note: it has to be on time with Triple osc's notes)
Step 8. Listen to that beautiful uncanny valley like singing you created

TLDR: a vocal synth controls Triple osc's frequencies using FX and pairing with Formant

Visual representation
![Uploading Poorly drawn.png…]()
is it poorly drawn? Yes, but you get the idea

the perfect name will be:
Formal Tongue

### Please search the issue tracker for existing feature requests before submitting your own.

- [X] I have searched all existing issues and confirmed that this is not a duplicate.

**Comment by 0hStormy** [comment]
The image didn't upload, but it's a cool idea.

**Comment by YesImAsh** [comment]
yeah hold on, gimme a sec
![Poorly drawn](https://github.com/LMMS/lmms/assets/172396809/ebaab054-d359-4ab1-b3ce-89b1a1b615a6)
